doc/users/colormapnorms.rst

```diff
@@ -0,0 +1,126 @@
+.. _colormapnorm-tutorial:
+
+Colormap Normaliztions 
+================================
+
+Objects that use colormaps usually linearly map the colors in the
+colormap from data values *vmin* to *vmax*.  For example::
+
+    pcm = ax.pcolormesh(x, y, Z, vmin=-1., vmax=1., cmap='RdBu_r')
+
+will map the data in *Z* linearly from -1 to +1, so *Z=0* will
+give a color at the center of the colormap *RdBu_r* (white in this
+case).  
+
+Matplotlib does this mapping in two steps, with a normalization from
+[0,1] occuring first, and then mapping onto the indices in the
+colormap. Normalizations are defined as part of
+:func:`matplotlib.colors` module.  The default normalization is
+:func:`matplotlib.colors.Normalize`.  The artists that map data to
+color pass the arguments *vmin* and *vmax* to
+:func:`matplotlib.colors.Normalize`. We can substnatiate the
+normalization and see what it returns.  In this case it returns 0.5::
+
+    norm = mpl.colors.Normalize(vmin=-1., vmax=1.)
```

Comment:
Sorry to be clueless.  I put:
```rst
. ipython::

   In [1]: import matplotlib as mpl

   In [2]: norm=mpl.colors.Normalize(vmin=-1.,vmax=1.)

   In [3]: norm(0.)
   Out[3]: 0.5
```
in the rst and the compile gives me:

```
/Users/jklymak/Dropbox/matplotlib/doc/api/backend_managers_api.rst:5: WARNING: autodoc: failed to import module u'matplotlib.backend_managers'; the following exception was raised:
Traceback (most recent call last):
  File "/Users/jklymak/anaconda/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
ImportError: No module named backend_managers
/Users/jklymak/Dropbox/matplotlib/doc/api/backend_tools_api.rst:5: WARNING: autodoc: failed to import module u'matplotlib.backend_tools'; the following exception was raised:
Traceback (most recent call last):
  File "/Users/jklymak/anaconda/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 385, in import_object
    __import__(self.modname)
ImportError: No module named backend_tools
/Users/jklymak/Dropbox/matplotlib/doc/users/colormapnorms.rst:24: ERROR: Unknown directive type "ipython-source".
```

and of course the resulting html doesn't have the code block.  Have I just mis-installed sphinx?